pom.xml

```diff
@@ -63,13 +63,6 @@
         -->
       </exclusions>
     </dependency>
-    <!-- to make it work in eclipse during development -->
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
```

Comment:
Besides functional tests covered by CI, I checked that this still works in `hpi:run` and also `java -jar jenkins.war`.